### PR TITLE
[19] [20] As a user, I can come back to Home screen after Complete screen

### PR DESCRIPTION
**Pull request by manh-t**
https://github.com/manh-t/flutter-survey/issues/19
https://github.com/manh-t/flutter-survey/issues/20

## What happened 👀

Display check done when submit survey successfully and user is automatically navigated back to Home screen after 2 secs.

## Proof Of Work 📹

**Android**

https://user-images.githubusercontent.com/60863885/120176881-fd130400-c231-11eb-9a16-bcf3120cac9b.mp4

**iOS**

https://user-images.githubusercontent.com/60863885/120180081-8f68d700-c235-11eb-9ac4-43ec424cc8b7.mp4

### CODE REVIEW

**Review by markgravity — APPROVED** [review]
LGTM 🤖

File: lib/pages/complete/complete_page.dart [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          width: 200,
+          height: 200,
+          child: FlareActor(
+            Assets.animCheckSuccess,
+            alignment: Alignment.center,
+            fit: BoxFit.contain,
+            animation: _animation,
+            callback: (animationName) {
+              if (animationName == _animation) {
+                Future.delayed(Duration(seconds: 1), () {
```
Comment (sleepylee) [inline lib/pages/complete/complete_page.dart:23]: I'm just curious if this 1s is the same as the animation time? 🤔  or it's just a delay before moving after the 2s of the animation? 😄

**Review by sleepylee — APPROVED** [review]
good to know `flare` 👍

File: lib/pages/complete/complete_page.dart (hunk 1 above)
Comment (manh-t) [reply lib/pages/complete/complete_page.dart:23]: @sleepylee this is just a delay before moving after the animation is completed 😁

**Pull request merged**